pkg/driver/node.go

```diff
@@ -47,7 +49,7 @@ func (d *driver) NodeStageVolume(_ context.Context, req *csi.NodeStageVolumeRequ
 	}
 
 	readOnly := false
-	if req.GetVolumeContext()["readOnly"] == "true" {
+	if volumeContext["readOnly"] == "true" {
```

Comment:
Can we use something like that here 
```
readOnly, ok := volumeContext["readOnly"]
readOnlyFlag := ok && readOnly == "true"
```
Also can we extract "readOnly" as a constant.